Lib/Mpl/brigand.hpp

```diff
@@ -520,6 +520,11 @@ namespace brigand
   template <typename L>
   using as_pair = wrap<L, pair_wrapper>;
 }
+/*********************************************************************
```

Comment:
Removing unused code is always great but I don't think we should take lwip into a count. What do you think @odinthenerd @chieltbest ?